### PR TITLE
Donation tooltips are now escaped properly

### LINKED ISSUE #5876: Donation tooltips should be escaped before rendering
## Details

Tooltips are used to display text within the Sequoia form template in the donation options. These can only be set by an admin, however they are not being properly escaped when rendered. This allows content to be rendered that shouldn't be.

## Acceptance Criteria

-   [x] Tooltips display without potential side effects

### PR DESCRIPTION
**Pull request by JasonTheAdams**
Resolves #5876 

## Description

This tackles the problem that tooltips weren't being escaped properly, allowing in side-effects. The issue was that the tooltip being rendered was simply grabbing the node text without escaping it, and applying it directly to the attribute as a string.

Now, the node element is created and constructed using the DOM functions, which handle escaping and such properly.

## Affects

Donation tooltip options

## Visuals

Image 2021-07-13 at 2 58 31 PM

Image 2021-07-13 at 2 59 04 PM

## Testing Instructions

Set the following as the Text for a Donation Option: `"onmouseover=alert(/OOPS/)//`, then hover over that donation option in the form. It **should now** open an alert.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@since` tags included in DocBlocks
-   [ ] Changes logged to the `Unreleased` section of `CHANGELOG.md`
-   [ ] Includes unit and/or end-to-end tests
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

**Comment by JasonTheAdams** [comment]
@knowler Can you please review this when you get a chance?

**Comment by JasonTheAdams** [comment]
@knowler Friendly reminder to review this when you get a chance.

File: src/Views/Form/Templates/Sequoia/assets/js/form.js [hunk 1]
```diff
@@ -218,7 +218,11 @@
 					const compare = position === 'before' ? symbol + value : value + symbol;
 					// Setup tooltip unless for custom donation level, or if level label matches value
 					if ( value !== 'custom' && text !== compare ) {
-						const wrap = `<span class="give-tooltip hint--top hint--bounce ${ text.length < 50 ? 'narrow' : '' }" style="width: 100%" aria-label="${ text.length < 50 ? text : text.substr( 0, 50 ) + '...' }" rel="tooltip"></span>`;
+						const wrap = document.createElement('span');
+						wrap.classList.add('give-tooltip', 'hint--top', 'hint--bounce', text.length < 50 ? 'narrow' : '');
+						wrap.style.width = '100%';
+						wrap.setAttribute('aria-label', text.length < 50 ? text : text.substr( 0, 50 ) + '...');
+						wrap.setAttribute('rel', 'tooltip');
```
Comment (knowler) [inline src/Views/Form/Templates/Sequoia/assets/js/form.js:225]: I know this was there before, but `rel="tooltip"` is non-standard. We should be able to get rid of it — I remember noticing this when working on FFM and it still worked without it.

```suggestion
```

**Review by knowler — CHANGES_REQUESTED** [review]
I have one request, otherwise this is good to go.

**Comment by JasonTheAdams** [comment]
@knowler Sounds good. Please approve and merge. 👍

**Review by knowler — APPROVED** [review]
Great work, @JasonTheAdams.

**Pull request merged**